app/views/staff/users/_form.html.haml

```diff
@@ -3,6 +3,12 @@
 
   = f.govuk_text_field :name
   = f.govuk_email_field :email
+  = f.govuk_collection_radio_buttons :role,
+      list_of_user_roles,
+      :id,
+      :name,
+      legend: { text: "Role" },
```

Comment:
`legend` and `hint_text` should be translated (they go in `config/locales/models_and_forms.yml`)

The [form builder documentation](https://govuk-form-builder.netlify.com/building-blocks/localisation/) explains how to do this.